src/Analyzer.TemplateProcessor.UnitTests/ArmTemplateProcessorTests.cs

```diff
@@ -479,12 +582,14 @@ public void CopyResourceDependants_DependsOnIsSubResource_CopiesToCorrectResourc
                   ""location"": ""westus"",
                   ""kind"": ""functionapp"",
                   ""properties"": { },
+                  ""dependsOn"": [ ],
                   ""resources"": [
                     {
                       ""apiVersion"": ""2015-08-01"",
                       ""name"": ""web"",
                       ""type"": ""sourcecontrols"",
-                      ""properties"": { }
+                      ""properties"": { },
+                      ""dependsOn"": [ ],
```

Comment:
Why is an empty `"dependsOn"` array needed in so many resources?

Reply:
See my response in your other comment "Why was this test removed?"

Reply:
It sounds like this property isn't actually necessary in the JSON then.  I guess I'm confused why it was added everywhere if it doesn't need to be specified.